### ISSUE TITLE
[Feature Request] File List Improvements 

### ISSUE THREAD

**Issue by giiutfff** [zhanghai/MaterialFiles#745]
File context menu:
Could remove the three dots on the right to save display room, alternatively put this menu up right while select only one file by tapping the icon, and display the menu directly if long press the file. ( the current implementation also looks awkward if the file is in the middle of the screen, cause it need to be swipe dowm/up to see the whole list)

Continuous selecting files:
Imagine this scenario, delete first 500 files in a folder with 1000 files. Yes it could be done, but needed a lot tap/swipe.
So to do this nicely, tap the first file, then long press (continuous selecting, like holding shift, but only in select mode otherwise will conflict with above mention ) the 500th file, then select delete. Easy peasy😁

File name sorting:
It's really nice to have an app actually care about sorting CJK characters.
But it really makes more sense to sort symbols first, then letters, kana, hangul, and finally Chinese character. Kinda like the reverse of the current implementation.

**Comment by SnoutBug** [comment]
I would like to see a little more separation between entries in the list view similar to the call history in the los phone app to increase readability.

Perhaps something like this?
![example](https://user-images.githubusercontent.com/71790678/164054463-469e694b-9e69-410d-8397-e0a23dd3f79e.svg)

Another thing i would love to see is dedicated icons for for certain folders similar to the ones you see in the Hamburger menu

**Comment by jersa** [comment]
> I would like to see a little more separation between entries in the list view similar to the call history in the los phone app to increase readability.

Came here to suggest the same thing and not surprised that someone has already requested it. A more compact file listing layout with less padding around each file entry would look better in my opinion and provide more information in less space. Maybe it could be implemented as part of a feature providing different view types, (list, details, thumbnail...) as in most of the many file manager applications I've used in the past. If I ever get around to installing Android Studio again I might submit a PR. 🙄

I like the app a lot. It is, without a doubt, the most promising candidate as of late in my often disappointing personal quest to find a reasonable, featureful, and pragmatic free software file manager for Android.

**Comment by rancidfrog** [comment]
The three dots button needs improvements, as when scrollbar is initiated, user is unable to access it until you wait for scrollbar to disappear.
- scrolling brings scroller on the right which overlaps three dot button, 
![](https://i.imgur.com/JzgNdSk.jpg) 
User is unable to click on three dots until you wait a moment for it to disappear
- only after scrollbar vanishes with time, can the user click on 3dots
![](https://i.imgur.com/lWLfu1d.jpg)

**Comment by theseanl** [comment]
I'd love to have an option to reduce file entries padding, currently the amount of information displayed on a screen is too low IMO.
